config/tomcat.yml

```diff
@@ -39,3 +39,6 @@ redis_store:
   database: 0
   timeout: 2000
   connection_pool_size: 2
+geode_store:
+  version: 0.+
+  repository_root: "https://tomcat-javabuildpack-spike.s3.amazonaws.com/geode-store"
```

Comment:
If the code really is Geode, and is all backed by an ASL repository, we could host it for you.